services/merak-topo/database/database.go

```diff
@@ -40,7 +41,8 @@ func ConnectDatabase() error {
 	})
 
 	if err := client.Ping(Ctx).Err(); err != nil {
-		return fmt.Errorf("ConnectDB: connect DB error %s", err.Error())
+		utils.Logger.Error("ConnectDB: connect DB error %s", err.Error())
```

Comment:
can consider to use error.Wrap, just in case of retry, so we can log error as warning in these cases.

Reply:
can save `ConnectDB:`.

Reply:
log should container parameters of the action.

Reply:
addressed, tks